tests/test_content.py

```diff
@@ -4,6 +4,7 @@
 import pytest
 
 import httpx
+from httpx._content import encode_json
```

Comment:
I've just noticed this accidentally snuck in a private import.

@BERRADA-Omar would you be up for refactoring these tests?

We can test this against public API using `httpx.Request()`. Eg...

```python
data = {...}
req = httpx.Request("POST", "https://www.example.com/", json=data)
assert req.content == ...
assert req.headers = ...
```

Reply:
Hello, i'll make the change.

Reply:
@tomchristie i see that this is already fixed ?

Reply:
Yes. Thanks, tho.